orte/mca/ess/hnp/ess_hnp_module.c

```diff
@@ -459,6 +459,8 @@ static int rte_init(void)
     node = OBJ_NEW(orte_node_t);
     node->name = strdup(orte_process_info.nodename);
     node->index = ORTE_PROC_MY_NAME->vpid;
+    node->slots = 0;
+    ORTE_FLAG_SET(node, ORTE_NODE_FLAG_SLOTS_GIVEN);
```

Comment:
I don't think you really want to do this - setting that flag means we won't sense the number of available slots by counting the number of cores. So basically you are "locking" mpirun itself to 0 slots. IIRC, the object constructor sets `slots` to zero - otherwise, this initializer is fine.

Reply:
Thanks - I think you're right and these lines aren't needed for the patch. Verifying..

Reply:
Updated.